### PR TITLE
ksd: Update bump to include CoreDNS as well

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:
Add the missing CoreDNS to KSD bump script.
In case we change the image on KSD, it will be reflected on CNAO as well.

Use the original manifest because the updated one is parameterized already,
and isn't split to yamls (so can't use yaml-utils).

**Special notes for your reviewer**:
Note that sed won't fail in case `CORE_DNS_IMAGE` isn't found,
we can improve it on the future, it isn't since it doesn't change a lot.
It is the same case for the rest of the images.
 
**Release note**:
```release-note
None
```

### CODE REVIEW

**Comment by oshoval** [comment]
/cc @AlonaKaplan 

Depends on your [PR](https://github.com/kubevirt/cluster-network-addons-operator/pull/1521)
Need to rebase once it is merged

**Comment by oshoval** [comment]
/hold 

Depends on the mentioned PR

**Comment by oshoval** [comment]
/cc @RamLavi @AlonaKaplan 

Ready for review
Thanks

**Review by RamLavi — APPROVED** [review]
/approve

@AlonaKaplan please give your lgtm

[update] - I see you already added it . never mind then 
/lgtm

**Comment by oshoval** [comment]
Thanks

**Pull request merged**
